IndividualSpreadFooting.tex

```diff
@@ -18,10 +18,10 @@ \section{Introduction} Clause 34 of the  \citetitle{IS2000} gives the
 
 \item 25\% excess pressure on edge of footing was allowed by the 
 \citetitle{IS1964} when a footing is eccentrically loaded. This 
-concession is withdrawn by the \citetitle{IS2000}, thereby adding to 
+concession is withdrawn by the ,\citetitle{IS2000} thereby adding to 
```

Comment:
What is right? Do you need comma in this line?
If yes, where do you want to place it?

Let me know. I can merge after correcting this.

Which one of the following do you need?  (I guess first one is right)

1. withdrawn by the, \citetitle{IS2000} thereby
2. withdrawn by the,\citetitle{IS2000} thereby